### ISSUE TITLE
When I trained my own video base on "activitynet_iter_30000_3fps.caffemodel",it stoped at some one iteration.

### ISSUE THREAD

**Issue by yp2005** [VisionLearningGroup/R-C3D#53]
The training data is my own, and the config file is default ,when I trained it 

rpn: num_positive 15
rpn: num_negative 49
I0306 13:32:56.740681 15943 accuracy_layer.cpp:96] Accuracy: 0.90625
I0306 13:32:56.741325 15943 accuracy_layer.cpp:101] Class 0 accuracy : 0.877551
I0306 13:32:56.741331 15943 accuracy_layer.cpp:101] Class 1 accuracy : 1
TRAIN
I0306 13:32:56.769326 15943 solver.cpp:228] Iteration 97, loss = 0.509901
I0306 13:32:56.769354 15943 solver.cpp:244]     Train net output #0: accuarcy = 0.956522
I0306 13:32:56.769363 15943 solver.cpp:244]     Train net output #1: loss_cls = 0.1252 (* 1 = 0.1252 loss)
I0306 13:32:56.769368 15943 solver.cpp:244]     Train net output #2: loss_twin = 0.0680313 (* 1 = 0.0680313 loss)
I0306 13:32:56.769371 15943 solver.cpp:244]     Train net output #3: rpn_accuarcy = 0.90625
I0306 13:32:56.769376 15943 solver.cpp:244]     Train net output #4: rpn_accuarcy_class = 0.877551
I0306 13:32:56.769378 15943 solver.cpp:244]     Train net output #5: rpn_accuarcy_class = 1
I0306 13:32:56.769382 15943 solver.cpp:244]     Train net output #6: rpn_cls_loss = 0.306048 (* 1 = 0.306048 loss)
I0306 13:32:56.769387 15943 solver.cpp:244]     Train net output #7: rpn_loss_twin = 0.0106217 (* 1 = 0.0106217 loss)
I0306 13:32:56.769392 15943 sgd_solver.cpp:106] Iteration 97, lr = 0.0001

just like this,it stoped here, then there is nothing more.
I wonder what's wrong with it,

**Comment by yp2005** [comment]
I tried to train on default dataset, the same problem happened.
